Dockerfile

```diff
@@ -35,18 +31,32 @@ RUN apt-get remove -y ncbi-blast+
 RUN wget -N ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.9.0/ncbi-blast-2.9.0+-1.x86_64.rpm
 RUN alien -i ncbi-blast-2.9.0+-1.x86_64.rpm
 RUN rm -rf ncbi-blast-2.9.0+-1.x86_64.rpm
-RUN bash -c "yes | cpan Bio::Perl"
+# The following line differs from the official instructions in that it actually works.
+RUN cpan List::Util
 WORKDIR /usr/local
 RUN git clone https://github.com/tseemann/prokka.git && /usr/local/prokka/bin/prokka --setupdb
 RUN ln -sf /usr/local/prokka/bin/prokka /usr/local/bin/prokka
 RUN prokka --version
 
+# In future we should append a layer that re-installs tbl2asn, which is a component
+# of blast that expires after 6 months, and is needed by prokka.  Uncomment this layer
+# when your jobs started failing in Prokka with error that tbl2asn is expired.
+# RUN wget ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux.tbl2asn.gz && \
+#    gunzip linux.tbl2asn.gz && \
+#    mv linux.tbl2asn tbl2asn && \
+#    chmod +x tbl2asn && \
+#    mv tbl2asn /usr/local/prokka/binaries/linux/
+
```

Comment:
For now we are sticking with prokka but in future we would consider using just prodigal because of this dependency --- which means in a few months this docker build will need to be repeated, with all corresponding fragility.